massa-consensus/src/settings.rs

```diff
@@ -11,6 +11,9 @@ use serde::{Deserialize, Serialize};
 use std::str::FromStr;
 use std::{default::Default, path::PathBuf, usize};
 
+// Consensus static parameters( defined by protocol used)
```

Comment:
wrong space placement + doc comment

Reply:
I do not know how to write a comment about all the following `const` difinitions? I can try a crate-level comment with `//!` on top of file!

Reply:
just /// doesn't work ?

Reply:
Maybe if you put every thing in a scope ? or a sub module

Reply:
It's dirty, I should do a re-export for this to work, and all that for a single comment not in generated doc, I do not think it worth it.

Reply:
Wrong space

Reply:
"parameters( defined" -> "parameters (defined"